contibuting.md

```diff
@@ -0,0 +1,38 @@
+# कंट्रिब्यूटिंग
+
+नमस्ते! हम रोमांचित हैं कि आप इस प्रोजेक्ट में योगदान करना चाहते हैं। इसे अच्छा बनाए रखने के लिए आपकी मदद जरूरी है। 
+
+यह डॉक्यूमेंट आपकी इस प्रोजेक्ट में योदान करने में सहायता करेगा ।
+
+# पुल्ल रिक्वेस्ट कैसे करें
+
+- इस [issue](https://github.com/gatsbyjs/gatsby-hi/issues/1) पर जाएं और पेज नाम कमेंट करें जिसको आप हिंदी में ट्रांसलेट करना चाहते हैं।
+- रिपॉजिटरी को fork करें।
+- Fork की हुई रिपॉजिटरी तो क्लोन करें (या ऑनलाइन Github एडिटर का इस्तेमाल करें)।
+- निचे दी गयी कमांड से एक नयी ब्रांच बनाएं।
+```
+git checkout -b नयी-ब्रांच-का-नाम
+```
+- जो फाइल आप ट्रांसलेट करना चाहते हैं उस फाइल में जाये और पूरी फाइल को हिंदी में ट्रांसलेट करें।
+- ट्रांसलेशन पूरा करने के बाद, सारे बदलाव को निचे दी गयी कमांड्स से `add`, `commit` और `push` करें।
+```
+git add फाइल-का-नाम 
+git commit -m "आपके बदलो के बारे में लिखें"
+git push origin नयी-ब्रांच-का-नाम
+```
+
+# रिव्यु प्रोसेस
+
+हर पुल्ल रिक्वेस्ट को २ चरण के रिव्यु को पार करना ज़रूरी है। रिव्यु के लिए हम निचे दी गयी बातों का पबातों का पालन करें।
+
+- सरे वर्गों का नाम अंग्रेज़ी में पुल्ल रिक्वेस्ट के विवरण में लिखें। जिसको रीवीउअर रिव्यु पूरा होने के बाद चेक मार्क करेगा।
+- किसी भी रीवीउअर के रीव्यु फीडबैक को रीसोलव मार्क न करें। रीवीउअर आपके फीडबैक फिक्सेस को देखने के बाद उसको रीसोलव मार्क करेगा।
+- आप फीडबैक के चंगेस करने के बाद उस फीडबैक पर कमेंट डालें या ये 👍 (इमोजी) लगा दें जिससे रीवीउअर को पता चल सके की आपने इस फीडबैक का फिक्स कर दिया है।
```

Comment:
के चंगेस => में परिवर्तन

Reply:
@SatyaRF Thanks you for reviewing. Fixed this.